source/telemetry.js

```diff
@@ -0,0 +1,31 @@
+const { Sequelize, QueryTypes } = require('sequelize');
+
+const version_helper = require('./version_helper.js')
+
+//// Log telemetry for Sequelize ORM.
+Sequelize.addHook('afterInit', async (connection) => {
```

Comment:
do you know how often this is executed? is it once for each connection? i think sequelize has a built in pool, so we might want to it to be once across all connections, like how the django adapter does https://github.com/cockroachdb/django-cockroachdb/pull/216

Reply:
I believe it is once after the Sequelize JS object is initialized itself.